.github/workflows/dict.json

```diff
@@ -0,0 +1,5 @@
+[
```

Comment:
Why a separate dictionary?

The project has its own config - https://github.com/hcodes/yaspeller/blob/master/.yaspellerrc

Reply:
oh, I didn't see that.

Also, I haven't implemented the action to support `.yaspellerrc` yet.